config/olm/addon-operator-metrics-service-tls.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
```

Comment:
This should not be required, please see my comment below. But you should double check my observations.